Realm/ObjectServerTests/SwiftObjectServerTests.swift

```diff
@@ -692,6 +692,137 @@ class SwiftObjectServerTests: SwiftSyncTestCase {
         XCTAssertEqual(ObjectiveCSupport.convert(object: Credentials.anonymous),
                        RLMCredentials.anonymous())
     }
+    // MARK: - Bundled Sync Realm
+
+    func testWriteCopySynchronizeData() {
+        do {
+            let user1 = try logInUser(for: basicCredentials())
+            if !isParent {
+                populateRealm(user: user1, partitionValue: #function)
+                return
+            }
+
+            // Wait for the child process to upload all the data.
+            // DELETE COMMENT: Does this need to be a child process? I doubt it.
+            executeChild()
```

Comment:
Does this need to be a different process?

Reply:
Executing this is a separate process isn't doing much by the way of testing `writeCopy`. What this is doing is just syncing the data from another user and then coping the Realm file.

Reply:
Correct, this can just be a multi user test.